stable_baselines3/common/utils.py

```diff
@@ -172,26 +172,29 @@ def configure_logger(
     tensorboard_log: Optional[str] = None,
     tb_log_name: str = "",
     reset_num_timesteps: bool = True,
-) -> None:
+) -> Logger:
     """
     Configure the logger's outputs.
 
     :param verbose: the verbosity level: 0 no output, 1 info, 2 debug
     :param tensorboard_log: the log location for tensorboard (if None, no logging)
     :param tb_log_name: tensorboard log
```

Comment:
Not new but this seems to be missing documentation for the reset_num_timesteps parameter?

(I like [darglint](https://github.com/terrencepreilly/darglint) for catching these errors.)

Reply:
nice catch